pkg/apiserver/api_installer.go

```diff
@@ -702,13 +711,32 @@ func addObjectParams(ws *restful.WebService, route *restful.RouteBuilder, obj ru
 					continue
 				}
 				desc := sf.Tag.Get("description")
-				route.Param(ws.QueryParameter(jsonName, desc).DataType(sf.Type.Name()))
+				route.Param(ws.QueryParameter(jsonName, desc).DataType(typeToJSON(sf.Type.Name())))
 			}
 		}
 	}
 	return nil
 }
 
+// TODO: this is incomplete, expand as needed.
+// Convert the name of a golang type to the name of a JSON type
+func typeToJSON(typeName string) string {
+	switch typeName {
+	case "bool":
+		return "boolean"
+	case "uint8", "int", "int32", "int64", "uint32", "uint64":
+		return "integer"
+	case "byte":
+		return "string"
+	case "float64", "float32":
+		return "number"
+	case "time/Time":
```

Comment:
@brendandburns @nikhiljindal Oops. This should have been `time.Time`, to match https://github.com/emicklei/go-restful/blob/master/swagger/model_builder.go#L279. However, I think we'd need `util.Time`, since we wrap time.Time.